docs/reference/ml/anomaly-detection/aggregations.asciidoc

```diff
@@ -25,7 +29,14 @@ You must also consider the interval of the date histogram aggregation carefully.
 The bucket span of your {anomaly-job} must be divisible by the value of the
 `calendar_interval` or `fixed_interval` in your aggregation (with no remainder).
 If you specify a `frequency` for your {dfeed}, it must also be divisible by this
-interval.
+interval. You cannot feed an {anomaly-job} with a date histogram with an 
+interval measured in months because the length of the month is not fixed. 
+{dfeeds-cap} tolerate weeks or smaller units.
+
+IMPORTANT: The name of the aggregation and the name of the field that the agg 
```

Comment:
> IMPORTANT: The name of the aggregation...

In general, I try to avoid putting two or more admonition blocks one after another.  Since the second one (the interval tip) seems to carry on subject matter from the second paragraph, I'd suggest putting this first one (important naming info) after the first paragraph.